pom.xml

```diff
@@ -5,16 +5,16 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>5.9</version>
+    <version>5.17</version>
     <relativePath />
   </parent>
 
   <properties>
-    <java.version>17</java.version>
-    <jenkins.version>2.502</jenkins.version>
+    <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
+    <jenkins.baseline>2.479</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
     <skipAnimalSniffer>false</skipAnimalSniffer>
     <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
-    <configuration-as-code.version>1932.v75cb_b_f1b_698d</configuration-as-code.version>
```

Comment:
No longer needed, version is taken from `io.jenkins.tools.bom:bom-${jenkins.baseline}.x`